### PR TITLE
fix: application can connect with the device

### LINKED ISSUE #834: Application sometimes doesn't detect the device
**Actual Behaviour**

Sometimes we have to unplug and plug back the device for the app to detect

**Expected Behaviour**

Whenever the device is attached, application should detect it.

**Steps to reproduce it**

1. Open app
2. Connect the device (This happens rarely but that is a big issue)

**LogCat for the issue**

> Not available

**Screenshots of the issue**

> Not available

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by CloudyPadmal**
Fixes #834 

Changes: 
- Added a menu item to connect the PSLab without manually plugging off and on.
- Update connection status on the menu bar by changing icons
- Added device initialization
- Added device ID to navigation menu

Screenshots for the change: 
recording

APK for testing:
[app-debug.apk.zip](https://github.com/fossasia/pslab-android/files/1997841/app-debug.apk.zip)

### CODE REVIEW

File: app/src/main/java/org/fossasia/pslab/activity/MainActivity.java [hunk 1]
```diff
@@ -76,6 +83,10 @@
     private Handler mHandler;
     private ScienceLabCommon mScienceLabCommon;
 
+    public boolean PSLabisConnected = false;
+
+    InitializationVariable initialisationStatus;
```
Comment (Vikum94) [inline app/src/main/java/org/fossasia/pslab/activity/MainActivity.java]: Can this be private?

Comment (CloudyPadmal) [reply app/src/main/java/org/fossasia/pslab/activity/MainActivity.java]: Hi @Vikum94 ! It is used in another fragment in the same package. So I left it in the package scope!

**Comment by CloudyPadmal** [comment]
Thank you @akarshan96 :+1:

**Comment by mariobehling** [comment]
Please resolve conflicts.

**Pull request merged**